components/KeywordLink/KeywordLink.js

```diff
@@ -0,0 +1,14 @@
+import Link from 'next/link';
+
+import HashIcon from '../../public/img/hashicon.svg';
+
+export const KeywordLink = (prop) => (
```

Comment:
```suggestion
export const KeywordLink = ({ children }) => (
```
We should prefer destructuring in the components, at all times. (when some pattern does not require otherwise)